CHANGELOG.md

```diff
@@ -113,7 +121,7 @@ Minor updates to dependencies to reduce the amount of compression libraries we d
   Some breaking cleanups to APIs to make them more ergonomic longterm
 
   - Many APIs that previously took Strings now take `AsRef<Utf8Path>`
-  - write_new_{all} now just takes a path to the file, instead of folder_path + name
+  - write*new*{all} now just takes a path to the file, instead of folder_path + name
```

Comment:
the heck

Reply:
Ugh autoformatting broke stuff again, let me fix that